.github/workflows/build.yml

```diff
@@ -84,6 +84,9 @@ jobs:
         path: hou
         key: vdb1-houdini18_0-${{ hashFiles('hou/hou.tar.gz') }}
         restore-keys: vdb1-houdini18_0-
+    - name: validate_houdini
+      if: github.repository_owner == 'AcademySoftwareFoundation'
+      run: ./ci/validate_houdini.sh
```

Comment:
can you just do this  in one line? `test -f "hou/hou.tar.gz"`

Reply:
I think so - I'll try that. It is simpler.